src/molecule/platforms.py

```diff
@@ -68,13 +68,19 @@ class Platforms(object):
               - child_group1
     """
 
-    def __init__(self, config, parallelize_platforms=False):
+    def __init__(self, config, parallelize_platforms=False, platform_name=None):
         """
         Initialize a new platform class and returns None.
 
         :param config: An instance of a Molecule config.
+        :param parallelize_platforms:
```

Comment:
Since you're adding this (although I'd argue that it's probably out of the scope), could you fill out the text description for this param too?

Reply:
It's fine. The description for parallelize_platforms is added in new commit